brainstormforce/astra
# AST-2401 - fix: issue with template kit alignment in editor

### Description


- Bug Fix: When the Full Width Contained layout is selected, the content inside the Container goes outside of the width - 
                 This happens only in Backend. On frontend, it works fine - https://share.bsf.io/BluW9Bbg

### Screenshots

https://d.pr/v/lxhqmK

### Types of changes


- Bug Fix: When the Full Width Contained layout is selected, the content inside the Container goes outside of the width - 
                 This happens only in Backend. On frontend, it works fine - https://share.bsf.io/BluW9Bbg



### How has this been tested?

- Create a page
- switch to compact mode 
- choose a template kit, it should look same in editor & frontend

### Checklist:
